libs/blocks/accordion/accordion.css

```diff
@@ -144,3 +144,82 @@ dl.accordion {
 .dark .accordion dt button:hover .accordion-icon::after {
   background: var(--color-gray-100);
 }
+
+/* Editorial Variation  */
+.accordion-media {
+  display:  none;
+}
+
+.accordion-media > div {
+  position: relative;
+  display: none;
+  animation-duration: .1s;
+  animation-name: slide-left;
+}
+
+/***** Slide to the Left *****/
+.slide-left {
+  animation: 1s slide-left;
+}
+@keyframes slide-left {
+  from {
+    margin-right: -100%;
+  }
+  to {
+    margin-right: 0%;
+  }
+}
+
+.accordion-media > div.expanded, 
+.accordion-media > div.expanded > img {
+  display: inline;
+  position: relative;
+  max-height: 525px;
```

Comment:
`max-height: 525px` and `width: auto` are also declared on L#189-190, I don't think you need them here as well. Everything declared for `accordion-media > div.expanded img` will also be applied to `accordion-media > div.expanded > img`

Reply:
You could probably delete 

```
.accordion-media > div.expanded > img {
  height: 525px;
}

.accordion-media > div.expanded img {
  max-height: 525px;
  width: auto;
}
```

and add those in to the parent rule 
```
.accordion-media > div.expanded, 
.accordion-media > div.expanded > img {
```